src/test/java/org/jenkinsci/plugins/matrixauth/integrations/ConfigurationAsCodeTest.java

```diff
@@ -0,0 +1,80 @@
+package org.jenkinsci.plugins.matrixauth.integrations;
+
+import com.cloudbees.hudson.plugins.folder.Folder;
+import com.cloudbees.hudson.plugins.folder.properties.AuthorizationMatrixProperty;
+import hudson.model.Computer;
+import hudson.model.Item;
+import hudson.security.AuthorizationStrategy;
+import hudson.security.HudsonPrivateSecurityRealm;
+import hudson.security.ProjectMatrixAuthorizationStrategy;
+import jenkins.model.Jenkins;
+import org.jenkinsci.plugins.matrixauth.AuthorizationMatrixNodeProperty;
+import org.jenkinsci.plugins.matrixauth.inheritance.InheritGlobalStrategy;
+import org.jenkinsci.plugins.matrixauth.inheritance.NonInheritingStrategy;
+import org.junit.Rule;
+import org.junit.Test;
+import io.jenkins.plugins.casc.misc.ConfiguredWithCode;
+import io.jenkins.plugins.casc.misc.JenkinsConfiguredWithCodeRule;
+import org.jvnet.hudson.test.LoggerRule;
+
+import static org.junit.Assert.*;
+
+public class ConfigurationAsCodeTest {
```

Comment:
export test?

Reply:
Perhaps if I decide I have way too much time. Until then I'll consider it something to be implemented once

> ![Screen Shot](https://user-images.githubusercontent.com/1831569/56500713-413c6200-650c-11e9-8953-850dfbdc1b88.png)


exists.

Reply:
https://github.com/jenkinsci/configuration-as-code-plugin/pull/851